identity.go

```diff
@@ -135,22 +134,17 @@ func UpgradeIdentity(b64Identity string) (*string, error) {
 
 
 func generateIdentity(config config, userIDString string) (*identity, error) {
-	generatedAppID := newAppId(config.AppSecret)
-
-	if !bytes.Equal(generatedAppID, config.AppID) {
+	if !bytes.Equal(newAppId(config.AppSecret), config.AppID) {
```

Comment:
Related to the other MR. Now that I see this, I think newAppId might not be a good name. It's more like getAppId(appSecret). `newAppId` suggests that you'll get a different one on each call, no?

Reply:
Indeed the name is misleading, will change this!